docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -1,14 +1,19 @@
-# Restricting access to specific prefixes in a S3 bucket
+# Restricting access to specific S3 bucket prefixes
 
 > Users will still have access to the full list of S3 objects,
 packages and logical keys inside of packages.
 
+> No trust relationship work is needed
+
 You can isolate user access to objects stored in specific S3 directories by
 defining an array of accessible prefixes in a custom IAM role or Amazon S3
 bucket policy.
 
-Create "custom" role or policy with these permissions:
+## Example permissions to limit access
+
+Create a "custom" role or policy with these permissions:
 
+<!-- markdownlint-disable -->
```

Comment:
Why do we need this?

Reply:
Are you referring to the json code block lable or what? Please propose a suggestion with the item removed so it's clear what we're talking about.

Reply:
No, I missed the line above with `<!-- markdownlint-disable -->`.